Update criteo.md

dbanda opened the pull request:
The new spec states that MergeTree does not take any parameters. Updating the Criteo example dataset doc to reflect this.

<!---
A technical comment, you are free to remove or leave it as it is when PR is created
The following categories are used in the next scripts, update them accordingly
utils/changelog/changelog.py
tests/ci/run_check.py
-->
### Changelog category (leave one):
- Documentation (changelog entry is not required)

dbanda committed:
Update criteo.md

The new spec states that MergeTree does not take any parameters. Updating the doc to reflect this.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/ClickHouse/ClickHouse?pullRequest=41958) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/ClickHouse/ClickHouse?pullRequest=41958) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/ClickHouse/ClickHouse?pullRequest=41958) it.</sub>

alexey-milovidov committed:
Update criteo.md

alexey-milovidov commented:
Ok. I've also removed index granularity, as defaults are good enough, and removed hashing of the second column of the primary key, as it can be only useful for sampling, and there is no SAMPLE BY.

dbanda commented:
> Ok. I've also removed index granularity, as defaults are good enough, and removed hashing of the second column of the primary key, as it can be only useful for sampling, and there is no SAMPLE BY.

makes sense. thanks!
